spring-cloud-modules/spring-cloud-contract/spring-cloud-contract-producer/pom.xml

```diff
@@ -46,8 +56,4 @@
         </plugins>
     </build>
 
```

Comment:
Not needed, interferes with parent's dependencyManagement spring-cloud-contract-verifier version